src/main/java/io/leonis/subra/game/data/Ball.java

```diff
@@ -50,6 +50,12 @@ default double getZ() {
     return this.getState().getMean().getDouble(3, 0);
   }
 
+  /**
+   * @return The XY-position of the {@link Ball}.
+   */
+  default INDArray getXY() {
```

Comment:
We might want to rename this to something more conventional like `getPosition` ? Although that's as specific as `getXY()`.

Reply:
there already is a `getPosition` which returns a `(x, y, r)`

Reply:
In that case; let's change this in the future if we feel like it.